docs/source/contributing/releasing.adoc

```diff
@@ -28,8 +21,38 @@ defined in your environment as _GITHUB_ACCESS_TOKEN_.
   For example, issues with a merged pull requests should be labeled as *resolution/done*.
 . Close milestone.
 
+[[auto-relase]]
+== Automated Release
+
+An automated release can be performed as:
+
+----
+$ export API_KEY=<api-key>
+$ export RELEASE_VERSION=<expected release version> # Eg : 1.0.0 (without 'v')
```

Comment:
`make release` was a bad target name. Call the target `trigger_release` or maybe even better `ci-release`. Obviously we need what happens in the current _release_ target, but nothing stops us from adding new targets, or in the case that we want to use `release` for triggering the CI release, we can also rename current _release_ into _do_release_. 

Bottom line, there needs to be two targets.